course.sql

```diff
@@ -0,0 +1,17 @@
+-- create table course
+CREATE TABLE course (
+  `id` INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  `name` VARCHAR(255) NOT NULL,
+  `day` VARCHAR(8) NOT NULL,
+  `time` VARCHAR(5) NOT NULL,
```

Comment:
it seems like there is a "TIME" data type in SQL : https://www.sqlservertutorial.net/sql-server-basics/sql-server-time/

Reply:
Thanks, I wasn't sure how to manage the time.  I correct, test and return as soon as possible.